ArduPlane/mode_auto.cpp

```diff
@@ -76,6 +76,16 @@ void ModeAuto::update()
         } else {
             plane.calc_throttle();
         }
+    } else if (plane.quadplane.in_transition()) {
```

Comment:
I think we can just treat transition like takeoff in Plane::set_servos_controlled()

Reply:
Do you mean this should be moved to set_servos_controlled?